.github/workflows/api-unit-test.yml

```diff
@@ -22,30 +22,13 @@ jobs:
     - name: Set up Go
       uses: actions/setup-go@v4
       with:
-        go-version: 1.19
-      
-    - name: Pull MongoDB
-      run: docker pull mongo
+        go-version: 1.21
```

Comment:
at go.mod is 1.20

Reply:
in go.mod what is shown is the minimum version of go required to compile, but any version above that can be used, 1.21 in this case